LEAF_Nexus/sources/Employee.php

```diff
@@ -1098,7 +1098,6 @@ public function lookupEmail($email)
                 LEFT JOIN {$this->tableName} USING (empUID)
                 WHERE indicatorID = 6
                     AND data = :email
-                    AND deleted = 0
```

Comment:
This should retain the deleted = 0 check. The default behavior must exclude inactive "deleted" employees.

If it's necessary to search for inactive employees by email, it needs to be made as an option.